.vscode/launch.json

```diff
@@ -10,6 +10,10 @@
                 "runserver",
                 "[::]:8000"
             ],
+            // Uncomment this to view debugger in /api requests
+            // "env": {
+            //     "USE_DEBUG_TOOLBAR": "True",
+            // },
```

Comment:
Since the launch configuration under version control, I would suggest either having this enabled by default or adding a separate profile for this (`Python: Django with Debug Toolbar`).

Reply:
@yattias I updated the launch configuration to enable the debug toolbar by default.